matrices.tex

```diff
@@ -25,6 +25,8 @@
 {x - sqrt(y*(x + y)): 1, x + sqrt(y*(x + y)): 1}
 \end{verbatim}
 
+%% TODO a Python list is not a dense representation...
+%% Is this supposed to say flat? - scopatz
```

Comment:
It isn't really flat either. Does this help? https://github.com/sympy/sympy-paper/pull/144/commits/332203b9ce19aeecf0b703be1f5284c85113b0f9